<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CoolReader</string>
  </property>
  <property name="windowIcon">
   <iconset resource="cr3res.qrc">
    <normaloff>:/icons/icons/cr3.png</normaloff>:/icons/icons/cr3.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>300</width>
     <height>300</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="sizeConstraint">
     <enum>QLayout::SetMinimumSize</enum>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="CR3View" name="view" native="true">
        <property name="contextMenuPolicy">
         <enum>Qt::CustomContextMenu</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollBar" name="scroll">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionRecentBooks"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionFileProperties"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionFindText"/>
    <addaction name="actionToggle_Pages_Scroll"/>
    <addaction name="actionMinimize"/>
    <addaction name="actionToggle_Full_Screen"/>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoom_Out"/>
    <addaction name="actionRotate"/>
    <addaction name="separator"/>
    <addaction name="actionTOC"/>
    <addaction name="actionShowBookmarksList"/>
    <addaction name="actionAddBookmark"/>
   </widget>
   <widget class="QMenu" name="menuNavigation">
    <property name="title">
     <string>Navigation</string>
    </property>
    <addaction name="actionNextPage"/>
    <addaction name="actionPrevPage"/>
    <addaction name="actionNextLine"/>
    <addaction name="actionPrevLine"/>
    <addaction name="actionFirstPage"/>
    <addaction name="actionLastPage"/>
    <addaction name="actionNextChapter"/>
    <addaction name="actionPrevChapter"/>
    <addaction name="separator"/>
    <addaction name="actionBack"/>
    <addaction name="actionForward"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAboutQT"/>
    <addaction name="actionAboutCoolReader"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuNavigation"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionOpen"/>
   <addaction name="actionRecentBooks"/>
   <addaction name="separator"/>
   <addaction name="actionTOC"/>
   <addaction name="separator"/>
   <addaction name="actionFindText"/>
   <addaction name="separator"/>
   <addaction name="actionPrevPage"/>
   <addaction name="actionNextPage"/>
   <addaction name="separator"/>
   <addaction name="actionZoom_In"/>
   <addaction name="actionZoom_Out"/>
   <addaction name="actionRotate"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/fileopen.png</normaloff>:/icons/action/icons/fileopen.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="toolTip">
    <string>Open file</string>
   </property>
   <property name="shortcut">
    <string>F3</string>
   </property>
  </action>
  <action name="actionToggle_Pages_Scroll">
   <property name="text">
    <string>Toggle Pages/Scroll</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/exit.png</normaloff>:/icons/action/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
  </action>
  <action name="actionNextPage">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/forward.png</normaloff>:/icons/action/icons/forward.png</iconset>
   </property>
   <property name="text">
    <string>Page Down</string>
   </property>
   <property name="toolTip">
    <string>Go to next page</string>
   </property>
   <property name="shortcut">
    <string>PgDown</string>
   </property>
  </action>
  <action name="actionPrevPage">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/back.png</normaloff>:/icons/action/icons/back.png</iconset>
   </property>
   <property name="text">
    <string>Page Up</string>
   </property>
   <property name="toolTip">
    <string>Back by page</string>
   </property>
   <property name="shortcut">
    <string>PgUp</string>
   </property>
  </action>
  <action name="actionNextLine">
   <property name="text">
    <string>Line Down</string>
   </property>
   <property name="toolTip">
    <string>Forward by one line</string>
   </property>
   <property name="shortcut">
    <string>Down</string>
   </property>
  </action>
  <action name="actionPrevLine">
   <property name="text">
    <string>Line Up</string>
   </property>
   <property name="toolTip">
    <string>Back by line</string>
   </property>
   <property name="shortcut">
    <string>Up</string>
   </property>
  </action>
  <action name="actionFirstPage">
   <property name="text">
    <string>First Page</string>
   </property>
   <property name="toolTip">
    <string>Go to first page</string>
   </property>
   <property name="shortcut">
    <string>Home</string>
   </property>
  </action>
  <action name="actionLastPage">
   <property name="text">
    <string>Last Page</string>
   </property>
   <property name="toolTip">
    <string>Go to last page</string>
   </property>
   <property name="shortcut">
    <string>End</string>
   </property>
  </action>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
   <property name="toolTip">
    <string>Back in navigation history</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
   <property name="toolTip">
    <string>Go to the next position in navigation history</string>
   </property>
   <property name="shortcut">
    <string>Shift+Backspace</string>
   </property>
  </action>
  <action name="actionNextChapter">
   <property name="text">
    <string>Next Chapter</string>
   </property>
   <property name="toolTip">
    <string>Go to next chapter</string>
   </property>
   <property name="shortcut">
    <string>Alt+Down</string>
   </property>
  </action>
  <action name="actionPrevChapter">
   <property name="text">
    <string>Previous Chapter</string>
   </property>
   <property name="toolTip">
    <string>Go to previous chapter</string>
   </property>
   <property name="shortcut">
    <string>Alt+Up</string>
   </property>
  </action>
  <action name="actionToggle_Full_Screen">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/window_fullscreen.png</normaloff>:/icons/action/icons/window_fullscreen.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Full Screen</string>
   </property>
   <property name="toolTip">
    <string>Toggle Full Screen mode</string>
   </property>
   <property name="shortcut">
    <string>Alt+Return</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/viewmag+.png</normaloff>:/icons/action/icons/viewmag+.png</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="toolTip">
    <string>Increase font size</string>
   </property>
   <property name="shortcut">
    <string>+</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/viewmag-.png</normaloff>:/icons/action/icons/viewmag-.png</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="toolTip">
    <string>Decrease font size</string>
   </property>
   <property name="shortcut">
    <string>-</string>
   </property>
  </action>
  <action name="actionTOC">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/view-table-of-contents-ltr.png</normaloff>:/icons/action/icons/view-table-of-contents-ltr.png</iconset>
   </property>
   <property name="text">
    <string>Table of Contents...</string>
   </property>
   <property name="toolTip">
    <string>Show table of contents</string>
   </property>
   <property name="shortcut">
    <string>F4</string>
   </property>
  </action>
  <action name="actionRecentBooks">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/folder_favorite.png</normaloff>:/icons/action/icons/folder_favorite.png</iconset>
   </property>
   <property name="text">
    <string>Recent Books</string>
   </property>
   <property name="toolTip">
    <string>Show recent books list</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/configure.png</normaloff>:/icons/action/icons/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings...</string>
   </property>
   <property name="toolTip">
    <string>Settings dialog</string>
   </property>
   <property name="shortcut">
    <string>F9</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/editcopy.png</normaloff>:/icons/action/icons/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy selected text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCopy2">
   <property name="text">
    <string>copy2</string>
   </property>
   <property name="toolTip">
    <string>Copy alternative shortcut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Ins</string>
   </property>
  </action>
  <action name="actionAboutQT">
   <property name="text">
    <string>About Qt</string>
   </property>
   <property name="toolTip">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAboutCoolReader">
   <property name="text">
    <string>About CoolReader</string>
   </property>
   <property name="toolTip">
    <string>About CoolReader</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAddBookmark">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/bookmark.png</normaloff>:/icons/action/icons/bookmark.png</iconset>
   </property>
   <property name="text">
    <string>Add Bookmark</string>
   </property>
   <property name="toolTip">
    <string>Add bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionShowBookmarksList">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/bookmark_folder.png</normaloff>:/icons/action/icons/bookmark_folder.png</iconset>
   </property>
   <property name="text">
    <string>Bookmark List...</string>
   </property>
   <property name="toolTip">
    <string>Show bookmarks list</string>
   </property>
   <property name="shortcut">
    <string>F6</string>
   </property>
  </action>
  <action name="actionFileProperties">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/player_playlist.png</normaloff>:/icons/action/icons/player_playlist.png</iconset>
   </property>
   <property name="text">
    <string>File Properties...</string>
   </property>
   <property name="toolTip">
    <string>Show file properties</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
  <action name="actionRotate">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/rotate_cw.png</normaloff>:/icons/action/icons/rotate_cw.png</iconset>
   </property>
   <property name="text">
    <string>Rotate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionFindText">
   <property name="icon">
    <iconset resource="cr3res.qrc">
     <normaloff>:/icons/action/icons/find.png</normaloff>:/icons/action/icons/find.png</iconset>
   </property>
   <property name="text">
    <string>Find text...</string>
   </property>
   <property name="toolTip">
    <string>Find text</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionToggleEditMode">
   <property name="text">
    <string>Toggle Edit Mode</string>
   </property>
   <property name="toolTip">
    <string>Toggle edit mode</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>Export</string>
   </property>
   <property name="toolTip">
    <string>Export document</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionNextPage2">
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="toolTip">
    <string>Next Page</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevPage2">
   <property name="text">
    <string>Previous Page</string>
   </property>
   <property name="toolTip">
    <string>Previous Page</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionNextPage3">
   <property name="text">
    <string>Next Page</string>
   </property>
   <property name="toolTip">
    <string>Next Page</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionNextSentence">
   <property name="text">
    <string>Next Sentence</string>
   </property>
   <property name="toolTip">
    <string>Move selection to next sentence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Right</string>
   </property>
  </action>
  <action name="actionPrevSentence">
   <property name="text">
    <string>actionPrevSentence</string>
   </property>
   <property name="toolTip">
    <string>Select orevious sentence</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Left</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CR3View</class>
   <extends>QWidget</extends>
   <header>cr3widget.h</header>
   <container>1</container>
   <slots>
    <signal>fileOpened()</signal>
    <slot>open()</slot>
    <slot>contextMenu(QPoint)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="cr3res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>view</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>MainWindowClass</receiver>
   <slot>contextMenu(QPoint)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>652</x>
     <y>32</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>contextMenu(QPoint)</slot>
 </slots>
</ui>
